OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -1087,15 +1080,15 @@ public void shouldAddDefaultButtonToAlertDialog() throws Exception {
       Button button = alert.getButton(AlertDialog.BUTTON_NEUTRAL);
       assertEquals(button.getText(), "Ok");
    }
-   
+
+   // TODO: Fails because of no appId being set because manifest placeholders removed
```

Comment:
Remove after rebase on other branch and tests are confirmed passing.

Reply:
Got it